src/RevSysCalls.cc

```diff
@@ -686,13 +737,29 @@ EcallStatus RevCore::ECALL_getdents64() {
   return EcallStatus::SUCCESS;
 }
 
+#if 1
+// https://man7.org/linux/man-pages/man2/lseek.2.html
+// 62, off_t rev_lseek( int fd, off_t offset, int whence)
+EcallStatus RevCore::ECALL_lseek() {
+  output->verbose(
+    CALL_INFO, 2, 0, "ECALL: lseek called by thread %" PRIu32 " on hart %" PRIu32 "\n", ActiveThreadID, HartToExecID
+  );
+  auto fd     = RegFile->GetX<int>( RevReg::a0 );
```

Comment:
Might be stupid but just want to check if @leekillough has any thoughts on the auto here? When you're explicitly getting an `<int>` does it make sense to do auto?

Reply:
I am fine with or without `auto`. We are using `<int>` so we're explicit about the RHS.

`int fd` is just as fine as `auto fd` 

When I prefer `auto` is when the type on the RHS is very long to type out (which `int` isn't), or when the RHS return type is declared somewhere else (like in a function being called), we don't control it here, and we don't care about the type for our uses here, like if we call a function which returns an integer type we don't care about and we only care if the return value is `0`:

```c++
auto val = some_func();
if( val != 0) {
}
```